tests/test_skyModel.py

```diff
@@ -65,10 +65,10 @@ def test_nexp_scaling(self):
         skymodel = desc.imsim.ESOSkyModel(obs_md, addNoise=False,
                                           addBackground=True)
         image_2 = galsim.Image(100, 100)
-        image_2 = skymodel.addNoiseAndBackground(image_2, photParams=photPars_2)
+        image_2 = skymodel.addNoiseAndBackground(image_2, 'R:4,2 S:1,0', photParams=photPars_2)
 
         image_1 = galsim.Image(100, 100)
-        image_1 = skymodel.addNoiseAndBackground(image_1, photParams=photPars_1)
+        image_1 = skymodel.addNoiseAndBackground(image_1, 'R:4,2 S:1,0', photParams=photPars_1)
```

Comment:
There should be a test comparing the sky background on two different chips which verifies that the median background levels are different.